composer.json

```diff
@@ -22,8 +22,8 @@
         }
     },
     "require": {
-        "php": ">=7.2",
-        "friendsofphp/php-cs-fixer": "^2.15.6",
+        "php": "^7.2|^8.0",
+        "friendsofphp/php-cs-fixer": "^2.17.1",
```

Comment:
`^2.15.6` should allow installing `2.17.1` already. Is there a reason the minimum version needs to be bumped to `2.17.1`?

Reply:
`^2.15.6` didn't support PHP 8 yet. IIRC, not upgrading the `php-cs-fixer` constraint resulted in errors.

Reply:
Ah ok. You may need to run `composer update matt-allan/laravel-code-style --with-dependencies` to get Composer to update php-cs-fixer to 2.17.1. If you try that it may work without having to bump the version here.

Reply:
It worked without changing anything on the Composer side. I probably wasn't clear, but I meant I was getting the error commented below.